.github/workflows/ci-pr-title.yaml

```diff
@@ -0,0 +1,93 @@
+name: CI Check Title
+
+on:
+  pull_request:
+    types: [opened, edited, synchronize, reopened]
+
+jobs:
+  title-lint:
+    name: Validate PR title
+    runs-on: [default]
+    steps:
+      - name: CI Check Title
+        uses: amannn/action-semantic-pull-request@v5
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        with:
+          wip: true
+          # Configure which types are allowed (newline-delimited).
+          # Default: https://github.com/commitizen/conventional-commit-types
+          types: |
+            build
+            chore
+            fix
+            feat
+            merge
+            publish
+            release
+            refactor
+            research
+            style
+            test
+          # Configure which scopes are allowed (newline-delimited).
+          # These are regex patterns auto-wrapped in `^ $`.
+          scopes: |
+            build
+            communicator
+            config
+            carbon
+            ci
+            core
+            deps
+            docs
+            doop
+            example
+            heureka
+            greenhouse            
+            infra
+            actions
+            juno
+            k8s
+            message-provider
+            npm
+            oauth
+            supernova
+            template
+            ui
+            utils
+            url-state-provider
+            version
+            ISSUE-\d+
```

Comment:
This list of scopes needs to be adapted to Greenhouse context.
I suggest we start with adding every CRD name + 'controller', 'webhook', 'greenhousectl' 
And removing Juno scopes such as `communicator, `carbon`, `doop` etc.

Reply:
fixed in [e8a19be](https://github.com/cloudoperators/greenhouse/pull/675/commits/e8a19bea9c8253bf60043ac6645ccf67f54a6086)